remotes/docker/config/hosts_test.go

```diff
@@ -297,6 +297,7 @@ func TestLoadCertFiles(t *testing.T) {
 
 func TestLocalhostHTTPFallback(t *testing.T) {
 	opts := HostOptions{
+		DefaultScheme: "http",
 		DefaultTLS: &tls.Config{
 			InsecureSkipVerify: true,
 		},
```

Comment:
Can we still test empty DefaultScheme too?

Reply:
Added more test cases